composer.json

```diff
@@ -154,7 +154,8 @@
     },
     "conflict": {
         "behat/transliterator": "< 1.3.0",
-        "doctrine/orm": "2.10.0 || 2.14.2 || >=2.16.0",
+        "doctrine/lexer": ">= 3.0.0",
```

Comment:
doctrine/lexer 3 not working with oro extensions: https://github.com/doctrine/orm/issues/11192 / https://github.com/oroinc/doctrine-extensions/pull/96